pkg/operator/ceph/csi/spec.go

```diff
@@ -102,6 +102,7 @@ type Param struct {
 	CSILogRotationMaxSize                    string
 	CSILogRotationPeriod                     string
 	CSILogFolder                             string
+	OperatorNamespace                        string
```

Comment:
How about we call it the csi driver namespace? Then it will be a more smooth transition to the CSI operator.
```suggestion
	CSIDriverNamespace                        string
```

Reply:
Done